tests/data/docker-compose.override-dev.yml

```diff
@@ -48,7 +48,7 @@ services:
     volumes:
       - ../snoop2:/opt/hoover/snoop:cached
       - ./gnupg:/opt/hoover/gnupg
-      - ./collections:/opt/hoover/collections
+      - ./collections/FL3:/opt/hoover/snoop/collection
```

Comment:
Maybe we shouldn't include references to "fl" in the public repo.

Reply:
These are just some initials...